src/Report/CwpStatsReport.php

```diff
@@ -15,6 +15,10 @@
 use SilverStripe\Subsites\Model\Subsite;
 use SilverStripe\Versioned\Versioned;
 
+if (!class_exists(Report::class)) {
+    return;
```

Comment:
Half of the reports featured below depend on `silverstripe/cms` being installed, which is no longer a requirement of this module. Do you think this should also be moved to `cwp/cwp` as with the TinyMCE stuff?

Reply:
Yep - let's move them to cwp/cwp

Reply:
https://github.com/silverstripe/cwp/pull/37